dockstore/dockstore-ui2
# DOCK-2533: fix regex errors in Edit Info dialog

**Description**
This PR fixes the regex errors showing on Firefox when opening the "Edit Info" dialog in the My Tools page. 

These errors show up on FireFox is because they use `v` mode - hence the trailing `/v` on the end of the regex patterns in the errors. This `v` flag is stricter than the `u` flag and required me to escape the characters. More information is [here](https://github.com/tc39/proposal-regexp-v-flag?tab=readme-ov-file#how-is-the-v-flag-different-from-the-u-flag)

Before:
![image](https://github.com/user-attachments/assets/0d64188e-d3a7-42a0-ae16-7753e4075fe8)

After:
![image](https://github.com/user-attachments/assets/d74e0d17-7f4b-4116-817f-7020b7bc12ff)

Verifies the regex still works:
![image](https://github.com/user-attachments/assets/cd73742f-87a2-46f6-a376-07726b4769a8)


**Review Instructions**
1. Go to My Tools
2. Select a tool, then go to versions tab
3. For a version, click Actions | Edit
4. Verify no regex errors are in the console.
5. Also edit the version with an invalid path to verify that the regex still works properly

**Issue**
https://ucsc-cgl.atlassian.net/browse/DOCK-2533

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!
